src/client/driver/driver.js

```diff
@@ -58,14 +66,15 @@ const CURRENT_IFRAME_ERROR_CTORS = {
 var hangingPromise = new Promise(noop);
 
 export default class Driver {
-    constructor (testRunId, selectorTimeout, heartbeatUrl, browserStatusUrl) {
+    constructor (testRunId, selectorTimeout, heartbeatUrl, browserStatusUrl, skipJsErrors) {
```

Comment:
We have too much args now, maybe `constructor(testRunId, communicationUrls, options)` instead?